### PR TITLE
MINOR FIX : Course Attachments Service Exception

### PR DESCRIPTION
**Pull request by imjane5**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CourseAttachments/CourseAttachmentServiceTests.Exceptions.Add.cs [hunk 1]
```diff
@@ -92,10 +93,14 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
         {
             // given
             CourseAttachment someCourseAttachment = CreateRandomCourseAttachment();
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CourseAttachments/CourseAttachmentServiceTests.Exceptions.Add.cs:96]: Remove this extra line

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CourseAttachments/CourseAttachmentServiceTests.Exceptions.Remove.cs [hunk 2]
```diff
@@ -152,7 +153,13 @@ public async Task ShouldThrowServiceExceptionOnRemoveWhenExceptionOccursAndLogIt
             Guid someAttachmentId = Guid.NewGuid();
             Guid someCourseId = Guid.NewGuid();
             var serviceException = new Exception();
-            var expectedCourseAttachmentException = new CourseAttachmentServiceException(serviceException);
+
+            var failedCourseAttachmentServiceException =
+                new FailedCourseAttachmentServiceException(serviceException);
+
+            var expectedCourseAttachmentException =
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CourseAttachments/CourseAttachmentServiceTests.Exceptions.Remove.cs:160]: epxectedCourseAttachmentServiceException

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Models/CoursesAttachments/Exceptions/FailedCourseAttachmentServiceException.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.CoursesAttachments.Exceptions
+{
+    public class FailedCourseAttachmentServiceException : Xeption
+    {
+        public FailedCourseAttachmentServiceException(Exception innerException)
+             : base(message: "Failed classroom service error occured, contact support.",
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/CoursesAttachments/Exceptions/FailedCourseAttachmentServiceException.cs:14]: Failed course attachment service....

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Models/CoursesAttachments/Exceptions/FailedCourseAttachmentServiceException.cs (hunk 3 above)
Comment (imjane5) [reply OtripleS.Web.Api/Models/CoursesAttachments/Exceptions/FailedCourseAttachmentServiceException.cs:14]: Changed the error message

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**